BraintreeCore/src/test/java/com/braintreepayments/api/ShopperInsightsClientUnitTest.kt

```diff
@@ -0,0 +1,44 @@
+package com.braintreepayments.api
+
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+import kotlin.test.assertIs
+
+/**
+ * Unit tests for BraintreeShopperInsightsClient.
```

Comment:
Fine to leave but for future work we don't typically add docs for tests (or internal methods unless extra explanation is required).